lib/css/components/_stacks-progress-bars.less

```diff
@@ -215,3 +215,56 @@
         }
     }
 }
+
+//  ===========================================================================
+//  $   CIRCULAR
+//  ---------------------------------------------------------------------------
+.s-progress__circular {
+    --s-progress-value: 0;
+    --s-circle-circumference: 88; // 2πr, r = 14.
```

Comment:
optional: If you wanna get real picky, you could do this math [in Less](http://lesscss.org/functions/#math-functions-pi), setting to a local less variable. It would also prevent attempts to override it on the client side. I don't foresee that happening, but I've seen less likely things before...